akka-docs/src/main/paradox/serialization.md

```diff
@@ -45,15 +45,17 @@ to disable a default serializer, see @ref:[Disabling the Java Serializer](remoti
 
 ### Enable additional bindings
 
-`akka.Done` is by default serialized by the Java serializer, add the following binding to avoid that.
-It's not enabled by default for compatibility reasons. 
+A few types in Akka are, for backwards-compatibility reasons, still serialized by using Java serializer by default.
+You can switch them add the following binding to avoid that, which will make them serialized using protocol buffers instead.
```

Comment:
"can switch them add" ?

You can switch them to using protocol buffers instead by adding the following bindings or set `akka.actor.allow-java-serialization=off`.

Reply:
ok